draft-ietf-quic-qpack.md

```diff
@@ -950,16 +974,16 @@ values that are not to be put at risk by compressing them (see Section 7.1 of
 {: title="Literal Header Field With Name Reference"}
 
 For entries in the static table or in the dynamic table with an absolute index
-less than or equal to Base Index, the header field name is represented using the
-relative index of that entry, which is represented as an integer with a 4-bit
-prefix (see Section 5.1 of [RFC7541]). The `S` bit indicates whether the
-reference is to the static (S=1) or dynamic (S=0) table.
+less than the Base, the header field name is represented using the relative
+index of that entry, which is represented as an integer with a 4-bit prefix (see
+Section 5.1 of [RFC7541]). The `S` bit indicates whether the reference is to the
+static (S=1) or dynamic (S=0) table.
 
 ### Literal Header Field With Post-Base Name Reference
 
-For entries in the dynamic table with an absolute index greater than Base Index,
+For entries in the dynamic table with an absolute index greater than the Base,
```

Comment:
If you change "less than or equal to" to "less than" above, maybe "greate than" here should be changed to "greater than or equal to"?